crates/bevy_app/src/app.rs

```diff
@@ -705,26 +712,27 @@ impl App {
         self
     }
 
-    /// Initialize a non-send resource with standard starting values by adding it to the [`World`]
+    /// Initialize a non-send [`Resource`] with standard starting values by adding it to the [`World`].
     ///
-    /// The resource must implement the [`FromWorld`] trait.
-    /// If the `Default` trait is implemented, the `FromWorld` trait will use
-    /// the `Default::default` method to initialize the resource.
+    /// The [`Resource`] must implement the [`FromWorld`] trait.
+    /// If the [`Default`] trait is implemented, the [`FromWorld`] trait will use
+    /// the [`Default::default`] method to initialize the [`Resource`].
     pub fn init_non_send_resource<R: 'static + FromWorld>(&mut self) -> &mut Self {
         self.world.init_non_send_resource::<R>();
         self
     }
 
     /// Sets the function that will be called when the app is run.
     ///
-    /// The runner function (`run_fn`) is called only once by [`App::run`]. If the
-    /// presence of a main loop in the app is desired, it is responsibility of the runner
+    /// The runner function `run_fn` is called only once by [`App::run`]. If the
+    /// presence of a main loop in the app is desired, it is the responsibility of the runner
     /// function to provide it.
     ///
-    /// The runner function is usually not set manually, but by Bevy integrated plugins
-    /// (e.g. winit plugin).
+    /// The runner function is usually not set manually, but by `bevy` integrated plugins
```

Comment:
Yeah okay, I dislike this style change. Can you please revert, and use "Bevy", rather than `bevy` consistently?